pypeman/channels.py

```diff
@@ -197,12 +200,14 @@ async def stop(self):
         - pypeman shuts down.
         - a channel is stopped (e.g. via the admin interface)
         """
+        self.logger.debug("Channel %s stopping ...", self)
```

Comment:
why `self.name` for starting, but `self` for stopping?

also `self` might cause pickle issues with socket loggerd

Reply:
Edit: I'll changed to self.name